### PR TITLE
[doc] fetch prop on * $vid

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
code pr #1486

### CODE REVIEW

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md [hunk 1]
```diff
@@ -7,9 +7,10 @@
 `FETCH PROP ON` 可返回节点的一系列属性，目前已支持一条语句返回多个节点属性。
 
 ```ngql
-FETCH PROP ON <tag_name> <vertex_id_list> [YIELD [DISTINCT] <return_list>]
+FETCH PROP ON { * | <tag_name> } <vertex_id_list> [YIELD [DISTINCT] <return_list>]
```
Comment (CPWstatic) [inline docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md]: FETCH PROP ON * <vertex_id> [YIELD [DISTINCT] <return_list>]

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md [hunk 2]
```diff
@@ -19,7 +22,9 @@ FETCH PROP ON <tag_name> <vertex_id_list> [YIELD [DISTINCT] <return_list>]
 ### 示例
 
 ```ngql
--- 如未指定 YIELD 字段，则返回节点 1 的所有属性
+-- 返回节点 1 的所有属性
+nebula> FETCH PROP ON * 1
```
Comment (CPWstatic) [inline docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md]: Should we add some blank line here?

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md [hunk 3]
```diff
@@ -24,16 +24,22 @@ FETCH PROP ON * <vertex_id>
...
+
 -- return property name and age of vertex id 1
 nebula> FETCH PROP ON player 1 YIELD player.name, player.age
+
 -- hash string to int64 as vertex id, fetch name and player
 nebula> FETCH PROP ON player hash(\"nebula\")  YIELD player.name, player.age
+
 -- find all neighbors of vertex 1 through edge e1. Then Get the neighbors' name and age.
 nebula> GO FROM 1 over e1 YIELD e1._dst AS id | FETCH PROP ON player $-.id YIELD player.name, player.age
+
```
Comment (wilsonyou) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md]: not sentence. It's statement

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md [hunk 4]
```diff
@@ -19,16 +22,24 @@ FETCH PROP ON <tag_name> <vertex_id_list> [YIELD [DISTINCT] <return_list>]
...
+
+-- 如未指定 YIELD 字段，则返回节点 1 tag 为 player 的所有属性
```
Comment (wilsonyou) [inline docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md]: 字段or关键词

File: docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md [hunk 5]
```diff
@@ -19,16 +22,24 @@ FETCH PROP ON <tag_name> <vertex_id_list> [YIELD [DISTINCT] <return_list>]
 ### Examples
 
 ```ngql
--- return all the properties of vertex id 1 if no yield field is given.
+-- return all the properties of vertex id 1
```
Comment (wilsonyou) [inline docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md]: Add semicolons to all queries to keep style consistent?

File: docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md (hunk 4 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/fetch-syntax.md]: they are the same here

**Pull request merged**